README.md

```diff
@@ -121,3 +124,15 @@ dfmt_keep_line_breaks | `true`, **`false`** | Keep existing line breaks if these
 * Braces - `{` and `}`
 * Brackets - `[` and `]`
 * Parenthesis / Parens  - `(` and `)`
+
+## Visual Studio integration
+**dfmt** can be configured as an external tool in Visual Studio (2017 and 2019). Choose [Tools]->[External Tools...], click [Add] and fill in the following fields:
+
+|                   |                                     |
+|-------------------|-------------------------------------|
+|Title:             | `d&fmt`                             |
+|Command:           | `C:\D\dmd2\windows\bin\dub.exe`     |
+|Arguments:         | `run dfmt -- --inplace $(ItemPath)` |
+|Initial Directory: | `$(ItemDir)`                        |
+
+Now Alt+T Alt+F will reformat the current file on disk. If you forgot to save the file first, VS will warn that the file changed on disk and ask if you want to discard unsaved changes; Reply No, save and try again.
```

Comment:
this keyboard shortcut (Alt-F) might be broken by localization. In German VS it works (text is `Ex&tras`) but I don't know if it's so convenient in all languages. You should add a note here that what this keyboard shortcut is doing is actually just navigating through the top menu by keyboard.

Reply:
also you don't need the Alt for the F key (even when not invoking the Tools menu using the keyboard)

Reply:
Ok, isn't the standard keybinding Ctrl + K + D? Could dfmt be registered to use that?

Reply:
at least not using this recommended way here, which is just running dfmt as external tool. I think using some VS extension it should be possible.